subworkflows/local/blobtools.nf

```diff
@@ -42,11 +48,23 @@ workflow BLOBTOOLS {
     ADD_SUMMARY_TO_METADATA (
       config_file
     )
+
+    //windowstats_tsv.view()
+    //window_dir = GetDirectory(windowstats_tsv)
+    //window_dir.view()
+
+    //tsv = TextToTsv.(blastp)
```

Comment:
Can you clean this up if it is not needed for future features?

Reply:
Yes I can't believe I left it in sorry 😞